alpha/apps/kaltura/lib/kCurrentContext.class.php

```diff
@@ -16,7 +16,7 @@ class kCurrentContext
 	public static $ks_object;
 	
 	/**
-	 * The specified language in which the request was made
+	 * language code (2char) of the returned object
```

Comment:
need to have validation that the 2 char code is valid for each language -> if not need to throw exception